tests/test_slogpy_console.py

```diff
@@ -0,0 +1,15 @@
+"""Import slogpy from slog"""
+from slogpy import slog
```

Comment:
This will not work as you think it does.  Please make sure to run the code before submitting a PR.

Reply:
@Sajal008 It looks like you addressed the issue.  In the future when you address an issue, please resolve the conversation.  It makes it easier to keep track of what is left to do.